Capstone/Assets/Game/Scripts/StateMachine/PlayerStates/PlayerFSMState_ATTACK.cs

```diff
@@ -32,17 +33,25 @@ public class PlayerFSMState_ATTACK : PlayerFSMState
         private Quaternion _targetRotation;
         private GameObject camera;
 
+        private Dictionary<SwordType, float> _animDurations = new Dictionary<SwordType, float>(); 
+
         public float Timer { get { return timer; } }
+        public Dictionary<SwordType, float> AnimDurations { get { return _animDurations; } }   
 
         public PlayerFSMState_ATTACK(Player player, PlayerInputsScript input, Animator animator, GameObject sword)
         {
             _player = player;
             _input = input;
             _animator = animator;
             _sword = sword;
-            //_sword.GetComponent<Rigidbody>().detectCollisions = false;
             id = PlayerCondition.F_Attacking;
-            //_sword.GetComponent<Rigidbody>().detectCollisions = false;
+
+            _animDurations.Add(SwordType.Quartz, animator.runtimeAnimatorController.animationClips.
+                Where(clip => clip.name == "Sword And Shield Slash 2").ToArray()[0].length / 1.5f);
```

Comment:
The 1.5f, 1f, and 2.0f. Do we want to put those as variables? Are they the different speeds of the animations? because if they are then we should probably have those either be the same as the speed set in the Animator inspector or just have them as general variables